pundix/chain.json

```diff
@@ -0,0 +1,173 @@
+          "darwin/arm64": "https://github.com/pundix/pundix/releases/download/v0.1.3/pundix_0.1.3_Darwin_arm64.tar.gz",
+          "windows/amd64": "https://github.com/pundix/pundix/releases/download/v0.1.3/pundix_0.1.3_Windows_x86_64.zip"
+        }
+      },
+      {
+        "name": "pxv2",
+        "tag": "v0.2.3",
+        "height": 6869100,
+        "recommended_version": "v0.2.3",
+        "compatible_versions": [
+          "v0.2.3"
+        ],
+        "binaries": {
+          "linux/amd64": "https://github.com/pundix/pundix/releases/download/v0.2.3/pundix_0.2.3_Linux_x86_64.tar.gz",
+          "linux/arm64": "https://github.com/pundix/pundix/releases/download/v0.2.3/pundix_0.2.3_Linux_arm64.tar.gz",
+          "darwin/amd64": "https://github.com/pundix/pundix/releases/download/v0.2.3/pundix_0.2.3_Darwin_x86_64.tar.gz",
+          "darwin/arm64": "https://github.com/pundix/pundix/releases/download/v0.2.3/pundix_0.2.3_Darwin_arm64.tar.gz",
+          "windows/amd64": "https://github.com/pundix/pundix/releases/download/v0.2.3/pundix_0.2.3_Windows_x86_64.zip"
+        }
+      }
+    ]
+  },
+  "peers": {
+    "seeds": [
+      {
+        "id": "78d3eb3f15a20ab1d567660d35776abe0dee71d0",
+        "address": "pundix-mainnet-seed-node-1.pundix.com:26656",
+        "provider": "Pundi X"
+      },
+      {
+        "id": "3c37c6c42dfd9094117549794299a62d49c122eb",
+        "address": "pundix-mainnet-seed-node-2.pundix.com:26656",
+        "provider": "Pundi X"
+      }
+    ],
+    "persistent_peers": [
+      {
+        "id": "8bd41ea9f8ba7cfee4d19887cab487cdfc1177f4",
+        "address": "pundix-mainnet-node-1.pundix.com:26656",
+        "provider": "Pundi X"
+      },
+      {
+        "id": "6c1738220234a5e1b3caf94403ecd651e9759952",
+        "address": "pundix-mainnet-node-2.pundix.com:26656",
+        "provider": "Pundi X"
+      },
+      {
+        "id": "23abe2346d40f82cf0606e47931e58752f8b9348",
+        "address": "pundix-mainnet-node-3.pundix.com:26656",
+        "provider": "Pundi X"
+      },
+      {
+        "id": "20d275af6d025be144765291db5337ea059cce18",
+        "address": "pundix-mainnet-node-4.pundix.com:26656",
+        "provider": "Pundi X"
+      },
+      {
+        "id": "47f97d7baf028ddfd3b223baab0fa062eae75310",
+        "address": "pundix-mainnet-node-5.pundix.com:26656",
+        "provider": "Pundi X"
+      }
+    ]
+  },
+  "apis": {
+    "rpc": [
+      {
+        "address": "https://px-json.pundix.com",
+        "provider": "Pundi X"
+      }
+    ],
+    "rest": [
+      {
+        "address": "https://px-rest.pundix.com",
+        "provider": "Pundi X"
+      }
+    ],
+    "grpc": [
+      {
+        "address": "https://px-grpc.pundix.com",
+        "provider": "Pundi X"
+      }
+    ]
+  },
+  "images": [
+    {
+      "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-chain-logo.png"
+    },
```

Comment:
Not required, but I'd recommend merging these two image objects into a single object since the images are visually identical when rescaled.